src/java.base/share/classes/sun/security/ssl/CipherSuite.java

```diff
@@ -972,7 +974,7 @@ static String nameOf(int id) {
             }
 
             boolean found = false;
-            for (CipherSuite cs : CipherSuite.values()) {
+            for (CipherSuite cs : ciphers) {
```

Comment:
If there is a static allowed cipher suite list,  the logic here could be simplified to use the allowed cipher suite list.  Then, the "ciphers" may be not needed any longer.